apps/examples/mediaplayer/mediaplayer_main.c

```diff
@@ -0,0 +1,111 @@
+/****************************************************************************
+ *
+ * Copyright 2016 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <dirent.h>
+#include <string.h>
+#include <apps/shell/tash.h>
+#include <mediaplayer/mediaplayer.h>
+
+static void usage(char *cmd)
+{
+	printf("Usage: %s [option] [filename1] [filename2]\n", cmd);
+	printf("\t-p: Playing\n");
+	printf("\t-r: Recording\n");
+	printf("\t-x: Playing 2 files(Mixing)\n");
+	printf("\n");
+}
+
+static int media_player_tash_cb(int argc, char **argv)
+{
+	media_t *music = NULL;
```

Comment:
could you please rename music & music_ to improve readability?

Reply:
Ok